libr/util/big.c

```diff
@@ -180,11 +181,11 @@ R_API char *r_big_to_hexstr(RNumBig *b) {
 	sprintf (&ret_str[i], R_BIG_SPRINTF_FORMAT_STR, b->array[j--]);
```

Comment:
dont use sprintf, use snprintf and dont use &ret_str[i], ret_str+i is more readable

Reply:
also make, i, j, z and k of type size_t